SU-SOE/stanford_soe_helper
# SOE-2742: change CSS to look for youtube and vimeo

# READY FOR REVIEW

# Summary
- This adds a CSS check for youtube or vimeo before applying a height of 400px to an iFrame.

# Needed By (TBD)
- Next release

# Criticality
- Client is eagerly awaiting this.
- Fixes broken stuff


# Steps to Test
- Switch to this branch
- May need to clear caches
- ## Verify: 

Here's an example of a video page:
/magazine/article/wind-farms-can-be-tough-sell-residents-what-can-be-done

![screen shot 2018-04-09 at 15 39 31](https://user-images.githubusercontent.com/284440/38526754-5de18cd4-3c0c-11e8-896d-6c9ec9b17b2c.png)

![screen shot 2018-04-09 at 15 39 01](https://user-images.githubusercontent.com/284440/38526709-3ce00632-3c0c-11e8-9929-06146c963322.png)


Here's an example of an audio page:
/magazine/article/debbie-senesky-developing-electronics-extremes-space

![screen shot 2018-04-09 at 15 41 34](https://user-images.githubusercontent.com/284440/38526816-9e2b8b96-3c0c-11e8-9402-52d12de9a607.png)

![screen shot 2018-04-09 at 15 41 07](https://user-images.githubusercontent.com/284440/38526825-a317216a-3c0c-11e8-96c6-b3069dd06d8b.png)


# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2742
https://stanfordits.atlassian.net/browse/SOE-2817

## Related PRs

## More Information

## Folks to notify
@minorwm


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
